scripts/setup

```diff
@@ -366,6 +366,11 @@ eval ${PREFIX} invenio utils runindex --raise-on-error
 info_msg "- Items: ${ITEMS} ${CREATE_LAZY} ${DONT_STOP}"
 eval ${PREFIX} invenio fixtures create --pid_type item --schema 'http://ils.rero.ch/schemas/items/item-v0.0.1.json' ${ITEMS} --append ${CREATE_LAZY} ${DONT_STOP}
 
```

Comment:
Why does the local fields are been created and indexed between items creation and items indexing ?

Reply:
done